modules/catalog/src/testFixtures/java/org/apache/ignite/internal/catalog/CatalogTestUtils.java

```diff
@@ -126,10 +124,8 @@ public void beforeNodeStop() {
             }
 
             @Override
-            public void stop() throws Exception {
-                super.stop();
-
-                metastore.stop();
+            public CompletableFuture<Void> stopAsync() {
+                return allOf(super.stopAsync(), metastore.stopAsync());
```

Comment:
Same as above. Basically there are plenty of places within tests where it's better to use best-effort stop logic.